frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/pipelineCreateRuns.cy.ts

```diff
@@ -630,6 +621,36 @@ describe('Pipeline create runs', () => {
       );
     });
 
+    it('creates a schedule with trigger type cron with whitespace', () => {
+      createScheduleRunCommonTest();
+      createSchedulePage.findScheduledRunTypeSelector().findDropdownItem('Cron').click();
+      createSchedulePage.findScheduledRunCron().fill('@every 5m ');
+      createSchedulePage
+        .mockCreateRecurringRun(projectName, mockPipelineVersion, createRecurringRunParams)
+        .as('createSchedule');
+      createSchedulePage.submit();
+
+      cy.wait('@createSchedule').then((interception) => {
+        expect(interception.request.body).to.eql({
+          display_name: 'New recurring run',
+          description: 'New recurring run description',
+          pipeline_version_reference: {
+            pipeline_id: 'test-pipeline',
+            pipeline_version_id: 'test-pipeline-version',
+          },
+          runtime_config: {
+            parameters: { min_max_scaler: false, neighbors: 1, standard_scaler: 'no' },
+          },
+          trigger: { cron_schedule: { cron: '@every 5m' } },
```

Comment:
Where is the test for '@every 5m '
The one that contains the space?

Reply:
@diegolovison , I have created two separate case 
the one with whitespace  is in this line https://github.com/opendatahub-io/odh-dashboard/pull/3061/files/3d1bd4b1eb2198f3dca928339229211e5bb11cf0#diff-ff1a31b964e7adebb99b1bee1eaa6d2678fd21025df88198dd7da5ade83aa8aaR627